daemon/containerd/progress.go

```diff
@@ -209,7 +210,7 @@ func (p *pullProgress) UpdateProgress(ctx context.Context, ongoing *jobs, out pr
 					Current: 1 + seconds,
 					Units:   "s",
 				})
-		} else if sn.Kind == snapshots.KindCommitted {
+		case snapshots.KindCommitted:
```

Comment:
Looks like this is an exhaustive list, so we can add a case for the remaining ones at the end; that way the `exhaustive` linter can warn us if other kinds are added in future that we don't handle yet; https://github.com/moby/moby/blob/fac076ccf96b748a43f5d988f43c237e0e4d6ae2/vendor/github.com/containerd/containerd/v2/core/snapshots/snapshotter.go#L46-L52

```go
case snapshots.KindUnknown, snapshots.KindView:
	// Ignore other snapshot kinds
```